src/wizards/generateProject/generationWizard.ts

```diff
@@ -202,6 +203,12 @@ export async function generateProjectWizard() {
   state.targetDir = await getTargetDirectory(state.artifactId);
 
   const projectGenState: ProjectGenState = state as ProjectGenState;
+  await sendTelemetry(VSCodeCommands.CREATE_PROJECT, {
+    status: CMD_SUCCEED_VALUE,
```

Comment:
status is set as succeeded even before we perform the actual project creation. So we'll never see any failure

Reply:
Right. I think I can create a separate helper that does the `vscode.openFolder` and reports the telemetry right before that. That should ensure we still report failure if something happens in the download/extraction of the project.

Reply:
@fbricon I've updated the PR to perform the telemetry reporting just before `openProject(..)` is called. There's just one issue :

If starting with a workspace that has no folders opened, and generating a Quarkus project from that, it still doesn't report the telemetry. I'm using `sendTelemetry(...).then(() => openProject(projectDir));` . I can see :

```
vscode-redhat-telemetry: Event received: quarkusTools.createProject
vscode-redhat-telemetry: Sending 'track' event with
{"userId":"...","event":"quarkusTools.createProject","properties":{"status":"succeeded","buildTool":"Maven","shouldGenerateCode":true,"extensions":"io.quarkus:quarkus-agroal,io.quarkiverse.amazonservices:quarkus-amazon-dynamodb,io.quarkiverse.amazonservices:quarkus-amazon-iam","extension_name":"redhat.vscode-quarkus","extension_version":"1.9.1","app_name":"Visual Studio Code","app_version":"1.63.2","app_kind":"Desktop","app_remote":false}...}
```
in the debug log, but the event never ends up recorded. Is there some part of the telemetry reporting that doesn't finish immediately and would get interrupted by the eventual `vscode.openFolder` call ?